src/App.tsx

```diff
@@ -1,47 +1,37 @@
-import React, { useEffect, useState } from 'react';
+import React, { useEffect } from 'react';
 import classNames from 'classnames';
 
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
 import './App.scss';
 
 import { PostsList } from './components/PostsList';
-import { PostDetails } from './components/PostDetails';
 import { UserSelector } from './components/UserSelector';
 import { Loader } from './components/Loader';
-import { getUserPosts } from './api/posts';
-import { User } from './types/User';
-import { Post } from './types/Post';
+import { useAppDispatch, useAppSelector } from './app/hooks';
+import * as postsActions from './features/posts';
+import { PostDetails } from './components/PostDetails';
+import { clearPost } from './features/selectedPost';
```

Comment:
The import statement for 'clearPost' is correct, but ensure that this action is properly defined and exported in './features/selectedPost'.